packages/sdk/src/wallet/base/Wallet.ts

```diff
@@ -49,9 +49,20 @@ export abstract class Wallet {
    * @returns Promise resolving to array of token balances with chain breakdown
    */
   async getBalance(): Promise<TokenBalance[]> {
+    // TEMPORARY - will use optimism token list eventually
     const tokenBalancePromises = Object.values(SUPPORTED_TOKENS).map(
-      async (token) => {
-        return fetchERC20Balance(this.chainManager, this.address, token)
+      async (tokenInfo) => {
+        // Convert TokenInfo to Asset format
+        const asset = {
+          address: tokenInfo.addresses,
+          metadata: {
+            decimals: tokenInfo.decimals,
+            name: tokenInfo.name,
+            symbol: tokenInfo.symbol,
+          },
+          type: tokenInfo.symbol === 'ETH' ? 'native' : 'erc20',
+        } as const
```

Comment:
a potential improvement is we just update the `SUPPORTED_TOKENS` to be type `Asset[]`, that way we dont have to do this conversion everywhere we want to use them, wdyt?

Reply:
made this change, though SUPPORTED_TOKENS is temporary and will be ripped out soon